sources/packages/api/src/app.controller.ts

```diff
@@ -1,11 +1,19 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, Req } from '@nestjs/common';
+import { AllowAnyRole, Public } from 'nest-keycloak-connect';
 import { AppService } from './app.service';
 @Controller()
 export class AppController {
   constructor(private readonly appService: AppService) {}
 
   @Get()
+  @Public()
   getHello(): string {
     return this.appService.getHello();
```

Comment:
Do we need to keep this around?

Reply:
I will remove in next PR related to this story.